src/main/kotlin/app/revanced/patches/youtube/layout/animated/AnimatedLikePatch.kt

```diff
@@ -11,7 +11,7 @@ import app.revanced.util.patch.BaseResourcePatch
 @Suppress("unused")
 object AnimatedLikePatch : BaseResourcePatch(
     name = "Hide double tap to like animations",
-    description = "Hide the like animations when double tap the screen in the Shorts player at compile time.",
+    description = "Removes the like animations when double-tapping the screen in the Shorts player at compile time.",
```

Comment:
Less ambiguous version: 

```suggestion
    description = "Removes, at compile time, the like animations that appear when double-tapping the screen in the Shorts player.",
```